test/test_fcntl.rs

```diff
@@ -561,3 +561,26 @@ mod test_posix_fallocate {
         }
     }
 }
+
+#[cfg(any(
+    target_os = "dragonfly",
+    target_os = "netbsd",
+    target_os = "macos",
+    target_os = "ios"
+))]
+mod test_apple_netbsd {
+    use nix::fcntl::*;
+    use std::os::unix::io::AsRawFd;
+    use std::path::PathBuf;
+    use tempfile::NamedTempFile;
+
+    #[test]
+    fn test_path() {
+        let tmp = NamedTempFile::new().unwrap();
+        let fd = tmp.as_raw_fd();
+        let mut path = PathBuf::new();
+        let res =
```

Comment:
This `res` variable seems to be unused, and could potentially cause a CI failure, we may want to use `let _ = ` instead or simply ignore it